packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -142,14 +142,18 @@ function flattenTree( input = {}, prefix, token ) {
 /**
  * Transform given style tree into a set of style declarations.
  *
- * @param {Object} blockStyles Block styles.
+ * @param {Object}  blockStyles Block styles.
  *
+ * @param {boolean} isRoot      Whether the styles apply to the root selector.
  * @return {Array} An array of style declarations.
  */
-function getStylesDeclarations( blockStyles = {} ) {
+function getStylesDeclarations( blockStyles = {}, isRoot = false ) {
```

Comment:
Ignorable observation: would it make sense to keep the selector arg consistent with how we're doing in the `compute_style_properties()` PHP method?

That is, passing the `selector` and doing the check inside this method?

```js
ROOT_BLOCK_SELECTOR === selector
```